### DISCUSSION TITLE
Error with Vue3 inertia form with v-bind and v-model togeter

### DISCUSSION THREAD

**Discussion by dr-roshyara** [inertiajs/inertia#962] [Help (Vue)]
Hallao 
I  am using Vue3 input form with Laravel and inertia . 
However now, it is not working. I can only use either v-model or v-bind in the input form. 
I could solve this way the problems in my previous projects . However it is showing now the follwoing error: 

VueCompilerError: Unnecessary value binding used alongside v-model. It will interfere with v-model's behavior.

````
<body>
<tr  v-for="(user, uIndx) in users  " :key="uIndx" >
 <td >     {{ uIndx+1 }} </td>
  <td >  {{user.name}} </td>
  <!-- email  -->
 <td >{{user.email}} </td>
 <!-- Edit button  -->
<td class="px-6 py-4">
 <form @submit.prevent="submit">
 <input type="text"  id="name" v-bind:value="user.name"  v-model="form.name" /> 
<button type="submit" >Bearbeiten  </button>
</form> 
</tr>
<script>
 import { reactive } from 'vue'
 import { Inertia } from '@inertiajs/inertia'
export default {
props:{
 users:  Array
},
   setup(){
       const form  =reactive({
          name :  "",
          email:  "",
       })
        function submit() {
              Inertia.post('/users', form)
        }
        return { form, submit }

   },
}
</script>
 </body>
`````
Can anyone please help me ? How can I bind the programatically determined value with inertia input form.

I tried to solve this way but still the value of form.name cant be detected automatically . it works only if I once click on the input field and then submit the form. Any other idea?
````
<form @submit.prevent="submit">
    <input :value="user.name" @input = "form.name = $event.target.value" />
        <button type="submit" class="px-4 py-1 text-sm text-white bg-blue-400 rounded">
          Bearbeiten
        </button>
</form> 
````

**Comment by ajnsn** [comment]
Hey @dr-roshyara 

Most of your question is not related to Inertia. In general you cannot bind a value together with `v-model` as `v-model` already binds the value.

It looks like you want to inline-edit user names inside a `v-for` loop. Here's a quick example of how this could be achieved by using a custom `@click` handler and passing the user object to a method. 

https://codepen.io/ajnsn/pen/rNzdNJg?editors=1011

As you are only editing one user, you should hen `patch` this respecitve users-endpoint including the user id, e.g. `/users/{user.id}`.  In such cases it's often easier to do an normal fetch/XHR request instead of using an Inertia request. You most likels find various examples by searching the other discussions.

Hope this makes sense! Feel free to try around with the Codepen a bit and let me know if there are any more questions.
